File: src/main/java-graphql/io/github/graphql/model/DeleteIssueCommentPayloadResponseProjection.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseField;
import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseProjection;

/**
 * Response projection for DeleteIssueCommentPayload
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:47+0800"
)
public class DeleteIssueCommentPayloadResponseProjection extends GraphQLResponseProjection {

    public DeleteIssueCommentPayloadResponseProjection() {
    }

    @Override
    public DeleteIssueCommentPayloadResponseProjection all$() {
        return all$(3);
    }

    @Override
    public DeleteIssueCommentPayloadResponseProjection all$(int maxDepth) {
        this.clientMutationId();
        this.typename();
        return this;
    }

    public DeleteIssueCommentPayloadResponseProjection clientMutationId() {
        return clientMutationId(null);
    }

    public DeleteIssueCommentPayloadResponseProjection clientMutationId(String alias) {
        fields.add(new GraphQLResponseField("clientMutationId").alias(alias));
        return this;
    }

    public DeleteIssueCommentPayloadResponseProjection typename() {
        return typename(null);
    }

    public DeleteIssueCommentPayloadResponseProjection typename(String alias) {
        fields.add(new GraphQLResponseField("__typename").alias(alias));
        return this;
    }


}

File: src/main/graphql/io/github/graphql/model/DeleteIssueCommentPayloadResponseProjection.kt
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseField
import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseProjection

/**
 * Response projection for DeleteIssueCommentPayload
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
open class DeleteIssueCommentPayloadResponseProjection : GraphQLResponseProjection() {

    override fun `all$`(): DeleteIssueCommentPayloadResponseProjection = `all$`(3)

    override fun `all$`(maxDepth: Int): DeleteIssueCommentPayloadResponseProjection {
        this.clientMutationId()
        this.typename()
        return this
    }

    fun clientMutationId(): DeleteIssueCommentPayloadResponseProjection = clientMutationId(null)

    fun clientMutationId(alias: String?): DeleteIssueCommentPayloadResponseProjection {
        fields.add(GraphQLResponseField("clientMutationId").alias(alias))
        return this
    }

    fun typename(): DeleteIssueCommentPayloadResponseProjection = typename(null)

    fun typename(alias: String?): DeleteIssueCommentPayloadResponseProjection {
        fields.add(GraphQLResponseField("__typename").alias(alias))
        return this
    }


}
